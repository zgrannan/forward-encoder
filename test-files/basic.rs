#[analyzer::run]
#[allow(dead_code)]
fn foo(x: u32) -> u32 {
    if x > 2 {
        3
    } else {
        4   
    }
}

fn main(){}
