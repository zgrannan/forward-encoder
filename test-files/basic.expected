Analyzing file test-files/basic.rs
Function foo():
Terminal block bb3: let _3 = _1 in
let _2 = Gt(move _3, const 2_u32) in
let _0 = if move _2 == Int(0) then const 4_u32 else const 3_u32 in
_0
